src/views/mixed-content/top-websites-with-mixed-content/index.js

```diff
@@ -0,0 +1,96 @@
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import React, {useState, useEffect} from 'react';
+import {api} from '../../../client';
+import {ResponsiveBar} from '@nivo/bar';
+import {ChartContainer} from './elements';
+
+const TopWebsitesWithMixedContent = () => {
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    api
+        .get('/api/mixed-content/top'+
```

Comment:
I'm curious why are these paths concatenated?

Reply:
If it is on one line, then it will surpass the max-length of a line.

Reply:
hahaha I didn't expect that

Is okay, then 👍